source/frontend/src/features/documents/ComposedDocument.ts

```diff
@@ -1,84 +1,99 @@
-import { DocumentRelationshipType } from '@/constants/documentRelationshipType';
-import {
-  Api_Document,
-  Api_DocumentMetadataUpdate,
-  Api_DocumentRelationship,
-  Api_DocumentType,
-  Api_DocumentUpdateRequest,
-  Api_DocumentUploadRequest,
-} from '@/models/api/Document';
-import {
-  Api_Storage_DocumentDetail,
-  Api_Storage_DocumentMetadata,
-  Api_Storage_DocumentTypeMetadataType,
-} from '@/models/api/DocumentStorage';
-import Api_TypeCode from '@/models/api/TypeCode';
+import { ApiGen_Base_CodeType } from '@/models/api/generated/ApiGen_Base_CodeType';
+import { ApiGen_CodeTypes_DocumentRelationType } from '@/models/api/generated/ApiGen_CodeTypes_DocumentRelationType';
+import { ApiGen_Concepts_Document } from '@/models/api/generated/ApiGen_Concepts_Document';
+import { ApiGen_Concepts_DocumentMetadataUpdate } from '@/models/api/generated/ApiGen_Concepts_DocumentMetadataUpdate';
+import { ApiGen_Concepts_DocumentRelationship } from '@/models/api/generated/ApiGen_Concepts_DocumentRelationship';
+import { ApiGen_Concepts_DocumentType } from '@/models/api/generated/ApiGen_Concepts_DocumentType';
+import { ApiGen_Mayan_DocumentDetail } from '@/models/api/generated/ApiGen_Mayan_DocumentDetail';
+import { ApiGen_Mayan_DocumentMetadata } from '@/models/api/generated/ApiGen_Mayan_DocumentMetadata';
+import { ApiGen_Mayan_DocumentTypeMetadataType } from '@/models/api/generated/ApiGen_Mayan_DocumentTypeMetadataType';
+import { ApiGen_Requests_DocumentUpdateRequest } from '@/models/api/generated/ApiGen_Requests_DocumentUpdateRequest';
+import { ApiGen_Requests_DocumentUploadRequest } from '@/models/api/generated/ApiGen_Requests_DocumentUploadRequest';
 import { UtcIsoDateTime } from '@/models/api/UtcIsoDateTime';
 
 export interface ComposedDocument {
-  mayanMetadata?: Api_Storage_DocumentMetadata[];
-  pimsDocumentRelationship?: Api_DocumentRelationship;
-  documentDetail?: Api_Storage_DocumentDetail;
+  mayanMetadata?: ApiGen_Mayan_DocumentMetadata[];
+  pimsDocumentRelationship?: ApiGen_Concepts_DocumentRelationship;
+  documentDetail?: ApiGen_Mayan_DocumentDetail;
   mayanFileId?: number;
 }
 
 export class DocumentRow {
   id?: number;
   mayanDocumentId: number | undefined;
-  documentType: Api_DocumentType | undefined;
-  statusTypeCode: Api_TypeCode<string> | undefined;
+  documentType: ApiGen_Concepts_DocumentType | undefined;
+  statusTypeCode: ApiGen_Base_CodeType<string> | undefined;
   fileName: string | undefined;
   isFileAvailable: boolean | undefined;
   appCreateTimestamp?: UtcIsoDateTime;
   appCreateUserid?: string;
   relationshipId: number | undefined;
-  relationshipType: DocumentRelationshipType | undefined;
-  parentId: number | undefined;
+  relationshipType: ApiGen_CodeTypes_DocumentRelationType | null = null;
+  parentId: string | undefined;
```

Comment:
this doesn't look related to the change to generated types, why is this necessary?